jest.config.ts

```diff
@@ -0,0 +1,61 @@
+import type { Config } from "jest";
+
+type ArrayElement<MyArray> = MyArray extends Array<infer T> ? T : never;
+
+const baseConfig: ArrayElement<NonNullable<Config["projects"]>> = {
+  roots: ["<rootDir>"],
+  testMatch: ["**/*.test.ts"],
+  // This combination of preset/transformIgnorePatterns enforces that both TS and
+  // JS files are transformed to CJS, and that the transform also applies to the
+  // dependencies in the node_modules, so that ESM-only dependencies are supported.
+  preset: "ts-jest",
+  // deliberately set to an empty array to allow including node_modules when transforming code:
+  transformIgnorePatterns: [],
+  modulePathIgnorePatterns: ["dist/"],
+  coveragePathIgnorePatterns: [".*.spec.ts", "dist/"],
+  clearMocks: true,
+  injectGlobals: false,
+};
+
+// Required by @peculiar/webcrypto, which comes from the polyfills
+// loaded in the setup file.
+process.env.OPENSSL_CONF = "/dev/null";
```

Comment:
This is only required in `solid-client-authn`, where we actually go in details about mocking crypto stuff.
```suggestion
```